src/views/navbar/style.js

```diff
@@ -346,3 +350,17 @@ export const MarkAllSeen = styled.span`
       props.isActive ? props.theme.brand.default : props.theme.text.alt};
   }
 `;
+
+// We make it a real link element because anchor links don’t work properly with React Router.
+// Ref: https://github.com/ReactTraining/react-router/issues/394.
+export const SkipLink = Tab.withComponent('a').extend`
+  grid-area: logo;
```

Comment:
Sorry for this question, but I'm not familiar with CSS grid, why this `logo` here?

Reply:
The areas of the grid are defined here:
https://github.com/Tom-Bonnike/spectrum/blob/eef68777727b8bda4c0bf908a8ef41e01fe203c9/src/views/navbar/style.js#L10

It’s just a way to slice your grids into named areas.

Because the skip link replaces the logo, I simply re-use that one.

Reply:
Ah, I see! Thanks!